app/components/v2/release_list_component.html.erb

```diff
@@ -230,6 +232,8 @@
     <% end %>
   <% elsif empty? %>
     <% container.with_empty_state do %>
+      <%#= render V2::ScheduledTrainComponent.new(train) %>
```

Comment:
unintentional?

Reply:
I'm thinking about whether to keep it or not. Not sure yet.